beetsplug/deezer.py

```diff
@@ -279,12 +279,27 @@ def _search_api(self, query_type, filters=None, keywords=""):
         if not query:
             return None
         self._log.debug(f"Searching {self.data_source} for '{query}'")
-        response = requests.get(
-            self.search_url + query_type,
-            params={"q": query},
-            timeout=10,
-        )
-        response.raise_for_status()
+        try:
+            response = requests.get(
+                self.search_url + query_type,
+                params={"q": query},
+                timeout=10,
+            )
+            response.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            self._log.error(
+                "Error fetching data from {} API\n Error: {}",
+                self.data_source,
+                e,
+            )
+            return None
+        except requests.exceptions.HTTPError as e:
+            self._log.error(
+                "HTTP error fetching data from {} API\n Error: {}",
+                self.data_source,
+                e,
+            )
+            return None
```

Comment:
Can't we just handle `RequestException`, given that the error message is more or less the same in both cases?
```suggestion
        except requests.exceptions.RequestException as e:
            self._log.error(
                "Error fetching data from {} API\n Error: {}",
                self.data_source,
                e,
            )
            return None
```